.storybook/stories/ThanksPageSingleVersion.stories.js

```diff
@@ -141,6 +141,15 @@ export const Badge = story({
 	myKivaEnabled: true,
 });
 
+export const BadgeLoansNotInPfpOptedIn = story({
```

Comment:
I think is actually `BadgeLoansInPfp`?

Reply:
@christian14b one more small change, I'd recommend renaming this story, since isn't the name I suggested more accurate (opposite the original name)?